python/src/sdf/pyWorld.cc

```diff
@@ -0,0 +1,173 @@
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "pyWorld.hh"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "sdf/Actor.hh"
+#include "sdf/Atmosphere.hh"
+#include "sdf/Frame.hh"
+#include "sdf/Joint.hh"
+#include "sdf/Light.hh"
+#include "sdf/Link.hh"
+#include "sdf/Model.hh"
+#include "sdf/Physics.hh"
+#include "sdf/World.hh"
+
+using namespace pybind11::literals;
+
+namespace sdf
+{
+// Inline bracket to help doxygen filtering.
+inline namespace SDF_VERSION_NAMESPACE {
+namespace python
+{
+/////////////////////////////////////////////////
+void defineWorld(pybind11::object module)
+{
+  // TODO(ahcorde)
+  //  - Added Scene methods
+  //  - Added Light methods
+  //  - Added Physics methods
+
+  pybind11::class_<sdf::World> geometryModule(module, "World");
+  geometryModule
+    .def(pybind11::init<>())
+    .def(pybind11::init<sdf::World>())
+    .def("validate_graphs", &sdf::World::ValidateGraphs,
+         "Check that the FrameAttachedToGraph and PoseRelativeToGraph "
+         "are valid.")
+    .def("name", &sdf::World::Name,
+         "Get the name of model.")
+    .def("set_name", &sdf::World::SetName,
+         "Set the name of model.")
+    .def("audio_device", &sdf::World::AudioDevice,
+         "Get the audio device name. The audio device can be used to "
+         "playback audio files. A value of \"default\" or an empty string "
+         "indicates that the system's default audio device should be used.")
+    .def("set_audio_device", &sdf::World::SetAudioDevice,
+         "Set the audio device name. See std::string AudioDevice() const "
+         "for more information.")
+    .def("wind_linear_velocity", &sdf::World::WindLinearVelocity,
+         "Get the wind linear velocity in the global/world coordinate "
+         "frame. Units are meters per second \f$(\frac{m}{s})\f$")
+    .def("set_wind_linear_velocity", &sdf::World::SetWindLinearVelocity,
+         "Set the wind linear velocity in the global/world coordinate "
+         "frame. Units are meters per second \f$(\frac{m}{s})\f$")
+    .def("gravity", &sdf::World::Gravity,
+         "Get the acceleration due to gravity. The default value is "
+         "Earth's standard gravity at sea level, which equals "
+         "[0, 0, -9.80665] \f$(\frac{m}{s^2})\f$")
+    .def("set_gravity", &sdf::World::SetGravity,
+         "Set the acceleration due to gravity. Units are meters per "
+         "second squared \f$(\frac{m}{s^2})\f$")
```

Comment:
I'm not sure that this LaTeX code `\f$(\frac{m}{s^2})\f` will render anywhere useful for python documentation. I would remove it from this and the 3 previous methods